src/data/BackendInterface.hpp

```diff
@@ -377,7 +377,7 @@ class BackendInterface {
      */
     std::optional<Blob>
     fetchLedgerObject(ripple::uint256 const& key, std::uint32_t sequence, boost::asio::yield_context yield) const;
-
+    
```

Comment:
I'm surprised clang-format is not screaming at this.. let's remove unnecessary whitespace leaving only `\n` as before